src/pages/ProductDetail/Info/InfoItem/InfoItem.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+import './InfoItem.scss';
+import ArrowButton from '../ArrowButton/ArrowButton';
+
+function InfoItem({ key, headline, text }) {
+  const [textView, setTextView] = useState(false);
+  const [slide, setSlide] = useState('');
+
+  function handleTextView() {
+    setTextView(!textView);
```

Comment:
함수형 업데이트 라는 키워드를 한 번 찾아보세요!
그 후 적용해보시면 좋겠습니다.

```suggestion
    setTextView(textView => !textView);
```

Reply:
넵!